ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/EntityTypeToggleGroup.tsx

```diff
@@ -2,51 +2,55 @@ import React from 'react';
 import { ToggleGroup, ToggleGroupItem } from '@patternfly/react-core';
 
 import useURLStringUnion from 'hooks/useURLStringUnion';
+import { NonEmptyArray } from 'utils/type.utils';
 import { entityTabValues, EntityTab } from './types';
 
 type EntityTabToggleGroupProps = {
-    entityContext?: EntityTab;
+    entityTabs?: Readonly<NonEmptyArray<EntityTab>>;
```

Comment:
This was added to support setting the default value in cases where `entityContext` was not `undefined`. e.g. If `entityContext="CVE"` the `useURLStringUnion` hook would still set the default parameter to "CVE" even though it was not a valid option.

Initially I left this prop alone and added an explicit `defaultEntity` prop, but ran into issues since the parent component using `EntityTabToggleGroup` had its own copy of the `useURLStringUnion` hook to read the current value and the parameters to each were different.

Doing it this way might make it easier to not forget to pass the same values to the hook and the component when declared as a const, but its probably subjective. 🤷‍♂️ 

The API difference is pretty much:
`<EntityTabToggleGroup entityContext="CVE" defaultEntity="Image" />`
vs
`<EntityTabToggleGroup entityTabs={["Image", "Deployment"] as const} />`

Any strong opinions either way?

Reply:
oh i see. i don't have a strong pref either way so that works for me!